dhcpv4/dhcpv4.go

```diff
@@ -262,12 +262,12 @@ func NewReplyFromRequest(request *DHCPv4, modifiers ...Modifier) (*DHCPv4, error
 }
 
 // NewReleaseFromACK creates a DHCPv4 Release message from ACK.
-// default Release message without any Modifer is created as following:
+// default Release message without any Modifiers is created as following:
```

Comment:
All these comment updates are unrelated to the RENEW support. Maybe they should be a different PR.

Reply:
Sure, I can remove them. Wanted to just improve the wording as I go, makes it easier to understand while figuring things out.